c-deps/libroach/db.cc

```diff
@@ -527,6 +527,40 @@ DBStatus DBEnvLinkFile(DBEngine* db, DBSlice oldname, DBSlice newname) {
   return db->EnvLinkFile(oldname, newname);
 }
 
+DBIterState DBCheckForAddSSTableKeyCollisions(DBIterator* existingIter, DBIterator* sstIter) {
```

Comment:
I think this wants a comment explaining what it does, what is/isn't a collision, etc.
also, I don't think there's anything specific to SSTs about it so the name could be more general, right?